subsys/debug/stack_size_analyzer/Kconfig

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig STACK_SIZE_ANALYZER
+	bool "Enable Stack size analyzer functionality"
+	select INIT_STACKS
+	select THREAD_MONITOR
+	help
+	  Enable stack size analyzer functionality and all the required modules.
+	  This module may be used to debug stack overflow or to find stacks
+	  which size may be optimized down.
+	  The module uses less resources than shell implementation and does not
+	  require a valid input to type the commands.
+
+if STACK_SIZE_ANALYZER
+module = STACK_SIZE_ANALYZER
+module-str = stack size analyzer
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+choice
+	prompt "Print mode"
+	default STACK_SIZE_ANALYZER_USE_LOG
+
+config STACK_SIZE_ANALYZER_USE_PRINTK
+	bool "Use printk function"
+	help
+	  Use kernel printk function to print stack size information.
+
+config STACK_SIZE_ANALYZER_USE_LOG
+	bool "Use logger output"
+	select LOG
+	help
+	  Use logger output to print stack size information.
+
+endchoice
+
+config STACK_SIZE_ANALYZER_AUTO
+	bool "Enable Stack size analyzer automatic printing"
```

Comment:
since it's already in stack size analyzer menu it should be shorter. Just `Enable automatic printing` is sufficient.